packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -129,6 +135,77 @@ angular
               }
             });
           })
+          .then(async () => {
+            $scope.user = await WucUser.getUser();
+          })
```

Comment:
**nitpick:** `async ... await` does not couple well with the AngularJS digest cycle. please use `$q`.